### PR TITLE
Add lr scheduler for auto_compress

### PR DESCRIPTION
**Pull request by zzjjay**

### CODE REVIEW

File: paddleslim/auto_compression/compressor.py [hunk 1]
```diff
@@ -489,7 +489,8 @@ def _start_train(self, train_program_info, test_program_info, strategy):
                 np_probs_float, = self._exe.run(train_program_info.program, \
                     feed=data, \
                     fetch_list=train_program_info.fetch_targets)
-
+                if not isinstance(train_program_info.learning_rate, float):
```
Comment (ceci3) [inline paddleslim/auto_compression/compressor.py:497]: 这里的learning_rate是什么形式呢？

Comment (ceci3) [inline paddleslim/auto_compression/compressor.py:497]: 这里的learning_rate是什么形式呢？

Comment (zzjjay) [reply paddleslim/auto_compression/compressor.py:497]: learning_rate 是 float 类型或 LRScheduler 类，当它为LRScheduler类时，需要调用step()更新学习率

File: demo/auto_compression/hyperparameter_tutorial.md [hunk 2]
```diff
@@ -116,7 +116,13 @@ TrainConfig:
   optim_args:
     weight_decay: 0.0005
 ```
-
+- 学习率设置衰减策略，如下所示：
+```yaml
+  learning_rate:
+    type: PiecewiseDecay # 学习率衰减策略类名
```
Comment (ceci3) [inline demo/auto_compression/hyperparameter_tutorial.md:122]: 学习率衰减策略类名 这个告诉下用户是从哪里看到要怎么设置，策略参数要怎么设置？

Comment (zzjjay) [reply demo/auto_compression/hyperparameter_tutorial.md:122]: done

**Review by ceci3 — APPROVED** [review]
